ER Circuit - connectivity capabilities

CHDAFNI-MSFT opened the pull request:
Not sure the goal or intent of the first 'ER best practices' bullet point...
By connecting Vnet 1 to ER Circuit 1... and Vnet 2 to ER Circuit 2... that alone doesn't peer them together unless on-premises is handling that routing. If so, that intention should be made clear.

Else what I'm proposing is a 'mesh' whereby a VNet has a connection to both ER circuits that way if 1 circuit goes down, there is still a connectivity path.

azurecla commented:
Hi __@gangsir__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

CHDAFNI-MSFT committed:
ER Circuit - connectivity capabilities

Not sure the goal or intent of the first 'ER best practices' bullet point...
By connecting Vnet 1 to ER Circuit 1... and Vnet 2 to ER Circuit 2... that alone doesn't peer them together unless on-premises is handling that routing. If so, that intention should be made clear.

Else what I'm proposing is a 'mesh' whereby a VNet has a connection to both ER circuits that way if 1 circuit goes down, there is still a connectivity path.
